common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
```

Comment:
I've moved this work to another PR for future work. It's a bit out of scope for this. This work only includes fixing the skip links to/within the transcript.